app/src/main/java/com/example/randomuserapplication/network/UserService.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.randomuserapplication.network
+
+import com.example.randomuserapplication.model.UserList
+import com.example.randomuserapplication.util.Constants.END_POINT
+import retrofit2.Response
+import retrofit2.http.GET
+
+interface UserService {
+    @GET(END_POINT)
```

Comment:
write out the endpoint here. don't extract it

Reply:
Oh ok